### PR TITLE
Add asset-to-path for IOC assets

### LINKED ISSUE #459: Add asset-to-path logic for all IOC assets
I _think_ this is the list. Look in the datalake for each of them, and add the needed goodness.

1125 - Kvitebjørn
1130 - Troll A
1138 - Valemon
1163 - Gudrun
1180 - Norne
1190 - Åsgard A
1191 - Åsgard B
1218 - Gina Krog
1219 - Aasta Hansteen
1230 - Visund
1755 - Grane
1775 - Troll B
1776 - Troll C

### PR DESCRIPTION
**Pull request by EinarJohnsen**
Path to existing IOC assets are added, including Johan Sverdrup. Will close #459

### CODE REVIEW

**Review by flikka — COMMENTED** [review]
Nice nice, let's just make a new issue for the two "missing" assets that you mention. And check if not Åsgård B has been forgotten?

File: gordo_components/data_provider/ncs_reader.py [hunk 1]
```diff
@@ -26,6 +26,14 @@ class NcsReader(GordoBaseDataProvider):
         "1218-gkr": "/raw/corporate/PI System Manager Sleipner/sensordata/1218-GKR",
         "1295-pera": "/raw/corporate/Aspen MS - IP21 Peregrino/sensordata/1295-PERA",
         "1110-gfa": "/raw/corporate/Aspen MS - IP21 Gullfaks A/sensordata/1110-GFA",
+        "1138-val": "/raw/corporate/PI System Operation Norway/sensordata/1138-VAL",
+        "1163-gdr": "/raw/corporate/PI System Manager Sleipner/sensordata/1163-GDR",
+        "1180-nor": "/raw/corporate/PI System Operation North/sensordata/1180-NOR",
+        "1190-asga": "/raw/corporate/PI System Operation North/sensordata/1190-ASGA",
```
[the remark this answers is not in the document]
Comment (EinarJohnsen) [reply gordo_components/data_provider/ncs_reader.py:26]: Åsgård B does already exist. The numbers could perhaps be sorted, which would make it easier to read.

**Comment by EinarJohnsen** [comment]
This has been tested, and seems to work with the new assets.

**Review by epa095 — APPROVED** [review]
LGTM if they work now (including jsv)

**Comment by milesgranger** [comment]
@EinarJohnsen Are you letting this sit and mature like a fine wine? :wine_glass: :older_man:

**Pull request merged**